scripts/kodiak/pmm_avellaneda.py

```
@@ -0,0 +1,445 @@
+        
+        # Immediately track the order
+        if order_id:
+            tracked_order = TrackedOrder(order_id=order_id)
+            self._tracked_orders[order_id] = tracked_order
+            # Update immediately if available
+            self._update_tracked_order(order_id)
+            self.logger().debug(f"Tracking order {order_id} for {order.order_side.name}")
+
+    def cancel_all_orders(self):
+        """Cancel all active orders, checking if they're actually open first."""
+        connector = self.connectors[self.config.exchange]
```

Flake8 (7.3.0)

[error] local variable 'connector' is assigned to but never used

(F841)